self-notes/peerFeedback.md

```diff
@@ -0,0 +1,7 @@
+1. Which do you prefer from eisenhowerSchema.sql file?
+
+email TEXT NOT NULL CHECK (position('@' IN email) > 1),
```

Comment:
Both format tends to add a constraint, for the sake of syntactic sugar😁 I would prefer the ln 3 format